scrapers/mmp_network.yml

```diff
@@ -0,0 +1,53 @@
+name: "PovBitch"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fakeshooting.com/video
+      - melonechallenge.com/video
```

Comment:
melonechallenge doesnt seem to use the same layout?

Reply:
I removed melonechallenge.com.